Makefile

```diff
@@ -9,7 +9,7 @@ else
        Q = @
 endif
 
-VERSION = $(shell git describe --dirty --tags)
+VERSION = $(shell git describe --dirty --tags || echo UNKNOWN)
```

Comment:
Why was this added?

Reply:
git Shallow copy can have no tags and this will fail. if it fails we add UNKNOWN so that travis is not broken here. at least that is my understanding

Reply:
Travis does a shallow checkout with a depth of 50. If none of the commits it retrieves have a tag, this `git describe` command will fail. We were hitting that problem this morning.

Reply:
The `make install` command started failing for this branch since travis by default uses a 50 commit depth in its pull. The last tag was more than 50 commits ago, so `make install` would not work correctly. This basically just makes it so that if no tag is found, we just use `UNKNOWN`

Reply:
lol that we all answered this at the same time.

Reply:
:+1:

Reply:
There is `--always` flag in `git describe`
https://git-scm.com/docs/git-describe#git-describe---always

Reply:
@shawn-hurley `git describe --dirty --tags --always`